scripts/build.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+const { join } = require("path");
+const mkdirp = require('mkdirp');
+const { readFileSync, writeFileSync } = require("fs");
+
+const rootPath = join(__dirname, '..');
+const destPath = join(rootPath, 'dist');
+const fontloaderFile = join(destPath, 'fontloader.js');
+
+function getFileContent(filePath) {
+    try {
+        const fileContent = readFileSync(filePath, 'utf8');
+        if (!fileContent) {
+            console.log(`${filePath} is empty`);
+            process.exit(1);
+        }
+        return fileContent;
+    } catch (err) {
+        console.log(err);
+        process.exit(1);
+    }
+    return '';
+}
+
+function wrapInIIFE(body) {
+    return `
+        ;(function() {
```

Comment:
Is this `;` needed?

Reply:
`;` is for safety when the file goes through a minifier like uglify. Everyone has a minifier in the build pipeline and this will safeguard it